packages/dashboard/src/components/resourceExplorer/getCurrentAssetProperties.ts

```diff
@@ -0,0 +1,80 @@
+import { AssetPropertySummary, IoTSiteWiseClient, ListAssetPropertiesCommand } from '@aws-sdk/client-iotsitewise';
+import { getEnvCredentials } from '../../../testing/getEnvCredentials';
+import { MaybeSiteWiseAssetTreeSessionInterface } from './types';
+import { HIERARCHY_ROOT_ID } from '.';
+import { REGION } from '../../../testing/siteWiseQueries';
+import { ExtendedPanelAssetSummary } from '.';
+import { DashboardMessages } from '../../messages';
+
+export const getCurrentAssetProperties = async (
+  provider: MaybeSiteWiseAssetTreeSessionInterface,
+  currentBranchId: string,
+  messageOverrides: DashboardMessages
+) => {
+  const assetPropertiesHeaderItem = {
+    id: messageOverrides.resourceExplorer.assetPropertiesHeader,
+    name: messageOverrides.resourceExplorer.assetPropertiesHeader,
+    isHeader: true,
+    isAssetProperty: false,
+    queryAssetsParam: [],
+  };
+
+  let client: IoTSiteWiseClient;
+  try {
+    client = new IoTSiteWiseClient({
+      region: REGION,
+      credentials: getEnvCredentials(),
+    });
+  } catch (e) {
+    console.log(e);
```

Comment:
What do you think of using console.warn here instead? Maybe even just throw an error? Thinking of how we'd use this for collecting metrics.